lib/msf/core/exploit/remote/kerberos/service_authenticator/base.rb

```diff
@@ -192,6 +204,7 @@ def authenticate(options = {})
       auth_context = authenticate_via_krb5_ccache_credential_tgs(options[:credential], options)
     else
       auth_context = authenticate_via_kdc(options)
+      auth_context = authenticate_via_krb5_ccache_credential_tgt(auth_context[:credential], options)
```

Comment:
This new call seems to break things for me:

```
msf6 auxiliary(admin/kerberos/get_ticket) > rerun verbose=true rhosts=192.168.123.13 domain=adf3.local user=Administrator password=p4$$w0rd action=GET_TGT
[*] Reloading module...
[*] Running module against 192.168.123.13

[*] 192.168.123.13:88 - Getting TGT for Administrator@adf3.local
[+] 192.168.123.13:88 - Received a valid TGT-Response
[*] 192.168.123.13:88 - TGT MIT Credential Cache saved to /Users/adfoster/.msf4/loot/20221202021234_default_192.168.123.13_mit.kerberos.cca_800619.bin
[-] Auxiliary aborted due to failure: unknown: Kerberos Error - KRB_ERR_GENERIC (60) - Generic error
[*] Auxiliary module execution completed
```

I don't think I'd expect an extra TGS-REQ call when running the module with GET_TGT

Reply:
Oh, too much rebases and I forgot to update this one. Good catch! Thanks!